FoodBookApp/UI/Views/BookmarksView.swift

```diff
@@ -27,6 +29,9 @@ struct BookmarksView: View {
                         try AuthService.shared.signOut()
                         showSignInView = true
                         
+                        // This should remain in the sign out
```

Comment:
I mean sign out view

Reply:
Wydm?

Reply:
That the next line `notify.cancelNotification(identifier: "lastReviewNotification")` is part of the sign out view and not the bookmarks view, because it will cancel notifications if a user signs out.

Reply:
I'll change this to a clearer comment (like the one above).